extension/content.js

```diff
@@ -390,6 +392,8 @@ document.addEventListener('DOMContentLoaded', () => {
 
 			if (pageDetect.isCommit()) {
 				addPatchDiffLinks();
+				addReactionParticipants.add(username);
+				addReactionParticipants.addListener(username);
 			}
```

Comment:
You could combine these in a single `if (pageDetect.isPR() || pageDetect.isIssue() || pageDetect.isCommit())`. I think it's more consistent, but it would also help with the fact `isCommit()` would return `true` for PR commits which match `isPR()` as well. So in some cases you would match both ifs and execute this twice. With a single if with 3 or expressions, you would execute it only once.